src/arch/host/include/arch/drivers/timer.h

```diff
@@ -7,8 +7,8 @@
 
 #ifdef __SOF_DRIVERS_TIMER_H__
 
-#ifndef __ARCH_TIMER_H__
```

Comment:
What is wrong with arch/host/include/arch/timer.h ?

Reply:
It was like that only for host. For dsp you have it in drivers.